toolstacks/php/drupal/customizing-settings-php.rst

```diff
@@ -19,9 +19,33 @@ Here is a default ``settings.php``:
     require_once($local_settings);
   }
 
+Any ``settings*.php`` file will be copied. It means you can use ``settings.dev.php`` and ``settings.staging.php``, then include the right one depending on environment variables.
+
+For example, you can load a development file for every development environment, by providing the ``drupal:load_development_overrides`` variable in Platform.sh UI, and using this code in the ``settings.php``:
+
+.. code-block:: php
+
+   <?php
```

Comment:
Might be best to provide the entire settings.php file? Or don't include the opening `<?php`